client/src/services/utilities.js

```diff
@@ -138,7 +139,7 @@ export function getNewFinancialData(data: NewEntityDetail): FinancialData {
   )
   return {
     ...pickBy(data.companyinfo, isValidValue),
-    ico: padIco(data.companyinfo.ico),
+    ico: padIco(data.companyinfo.ico) || '',
```

Comment:
What do you think about moving this `|| ''` to `padIco` function? Would it break the use in `extractIco`? This condition here seems pretty random.

Reply:
Yes, I should have added a comment - here I never expect null, `company.ico` must never be empty. But `padIco` (used also by `extractIco`) expects it may be null.
I'll check whether it breaks and move the condition there (with a note to remove it when old `extractIco` is no longer used)